tests/win_bindings.rs

```diff
@@ -11,5 +11,15 @@ fn gen_bindings() -> io::Result<()> {
     ];
 
     let bindings = windows_bindgen::standalone(&apis);
-    fs::write("src/offset/local/win_bindings.rs", format!("#![allow(unreachable_pub)]\n{bindings}"))
+
+    let path = "src/offset/local/win_bindings.rs";
+    let existing = fs::read_to_string(path)?;
+    // Skip the first two lines when comparing:
+    // - We add `#![allow(unreachable_pub)]` as a first line.
+    // - Rustfmt removes the empty line after the comment.
```

Comment:
I think we should make sure the code is formatted before it's written, so we can compare directly (and don't need to format as a separate stuff when we need to commit a new version).

Can you run `rustfmt` against the generated file? Or use the `prettyplease` crate, if it yields the same formatting for this code?